test/test-lib-nodejs.sh

```diff
@@ -519,6 +520,18 @@ function test_safe_logging() {
   fi
 }
 
+function save_test_result() {
```

Comment:
Why you are creating a new function? The variables can be a part of `run_all_tests`. I don't follow.
What about moving the function into `container-common-scripts` so we can use it generally.

Reply:
I have created this new function, because the `run_all_tests` is specific for nodejs container tests. In openshift tests we do not need `$CLIENT_LIST` and run client TCs in a specific way.
But it is true, that it can be written differently. Moving a more generic version of `run_all_tests` to `container-common-scripts` seems like a good idea, I will try it.

Reply:
PR for ccontainer-common-scripts available here: https://github.com/sclorg/container-common-scripts/pull/229